draft-ietf-lamps-csr-attestation.md

```diff
@@ -115,12 +115,12 @@ informative:
 
 --- abstract
 
-A PKI end entity requesting a certificate from a Certification Authority (CA) may wish to offer trustworthy claims about the platform generating the certification request and the environment associated with the corresponding private key, such as whether the private key resides on a hardware security module.
+A Public Key Infrastructure (PKI) end entity requesting a certificate from a Certification Authority (CA) may wish to offer trustworthy claims about the platform generating the certification request and the environment associated with the corresponding private key, such as whether the private key resides on a hardware security module.
```

Comment:
All other abbreviations were introduced before using them. So I think it would be reasonable to do the same with the term PKI.